config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'welcome/index'
```

Comment:
If we decide to remove the Welcome controller, I guess these routes are not needed right?

Reply:
welcome controller removed